flags/src/main/java/com/yahoo/vespa/flags/Flags.java

```diff
@@ -122,11 +122,17 @@
             "Takes effect on the next run of RetiredExpirer.",
             HOSTNAME);
 
-    public static final UnboundListFlag<PreprovisionCapacity> PREPROVISION_CAPACITY = defineListFlag(
-            "preprovision-capacity", List.of(), PreprovisionCapacity.class,
-            "List of node resources and their count that should be present in zone to receive new deployments. When a " +
-            "preprovisioned is taken, new will be provisioned within next iteration of maintainer.",
-            "Takes effect on next iteration of HostProvisionMaintainer.");
+    public static final UnboundListFlag<HostCapacity> PREPROVISION_CAPACITY = defineListFlag(
+            "preprovision-capacity", List.of(), HostCapacity.class,
+            "List of node resources and their count that should be unallocated in zone to receive new deployments. When a " +
+            "preprovisioned host is taken, another one will be provisioned within next iteration of maintainer.",
+            "Takes effect on next iteration of DynamicProvisioningMaintainer.");
+
+    public static final UnboundListFlag<HostCapacity> EXACT_PROVISION_CAPACITY = defineListFlag(
```

Comment:
I'm not sure about introducing another effectively permanent flag, especially when these 2 flags are mutually exclusive...

Reply:
Agree. Do you have something in mind? I suppose the other flag could be reused, but then we overload its meaning.

Reply:
> Do you have something in mind? 

I was hoping we could reuse the other flag. It's not ideal, but I think in the long run it will be less confusing than having 2 flags and needing to validate the state (only 1 of them should be set at a given time).